src/python/tests/core/test_scheduler_get_tasks.py

```diff
@@ -25,29 +25,31 @@
 14. LIMIT  + OFFSET + WHERE + SORT
 """
 
-import aiopg
 import asyncio
 import datetime
 import os
 import random
-import sqlalchemy
-import sqlalchemy.dialects.postgresql
+import signal
+import sys
 import time
 import uuid
-
+import aiopg
 import pytest
-
-from foglamp.core.scheduler import (Scheduler, IntervalSchedule, ScheduleNotFoundError, Task,
-                                    Schedule, TimedSchedule, ManualSchedule, StartUpSchedule, Where)
-
-
-__author__ = "Terris Linenbach"
+import sqlalchemy
+import sqlalchemy.dialects.postgresql
+from foglamp.core.scheduler.scheduler import Scheduler
+from foglamp.core.scheduler.entities import Task
+from foglamp.core.server import Server
+from foglamp.core.service_registry.instance import Service
+from foglamp.storage.exceptions import *
+from foglamp.storage.storage import Storage
+
+__author__ = "Terris Linenbach, Amarendra K Sinha"
 __copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
 __license__ = "Apache 2.0"
 __version__ = "${VERSION}"
 
 _CONNECTION_STRING = "dbname='foglamp'"
-
 _TASKS_TABLE = sqlalchemy.Table('tasks', sqlalchemy.MetaData(),
```

Comment:
As with the other comments on the test I think we must move away form this. For now lets mark this with a TODO and tackle this when we address testing in a future sprint.